ballista/executor/src/cpu_bound_executor.rs

```diff
@@ -146,7 +146,7 @@ impl DedicatedExecutor {
         let job = Box::pin(async move {
             let task_output = task.await;
             if tx.send(task_output).is_err() {
-                warn!("Spawned task output ignored: receiver dropped");
+                debug!("Spawned task output ignored: receiver dropped");
```

Comment:
Not really causing any noise but annoying when searching for `WARN`ings and there are a million of these to wade through...

Reply:
💯